docs/deploy-development-rp.md

```diff
@@ -216,6 +216,15 @@
   curl -X GET -k "https://localhost:8443/admin/subscriptions/$AZURE_SUBSCRIPTION_ID/resourceGroups/$RESOURCEGROUP/providers/Microsoft.RedHatOpenShift/openShiftClusters/$CLUSTER/kubernetespodlogs?podname=$POD&namespace=$NAMESPACE&container=$CONTAINER"
   ```
 
+## OpenShift Cluster Manager (OCM) Configuration API Actions
+
+* Create a new OCM configuration
+  * You can find example payloads in the projects `./hack/ocm` folder.
+
+  ```bash
+  curl -X PUT -k "https://localhost:8443/subscriptions/$AZURE_SUBSCRIPTION_ID/resourceGroups/$RESOURCEGROUP/providers/Microsoft.RedHatOpenShift/openShiftClusters/$CLUSTER/syncsets/mySyncSet?api-version=2022-09-04" --header "Content-Type: application/json" -d @./hack/ocm/syncset.json
```

Comment:
If a base64 PUT is supported, should we document it here alongside the JSON?

Reply:
I'll add it to the documentation, but the ordering of shipping the API before we have an API contract with OCM results in this - unsure requirements and inputs. Technically I should remove the json example and just use the b64 one as thats how we have written our swagger spec.